packages/gradbench/src/App.css

```
@@ -0,0 +1,117 @@
+:root {
+  --dark_blue: #336699;
+  --blue_grey: #3e4756;
+}
+
+body {
```

Maybe get rid of the margin around the header at the top of the page?

```suggestion
body {
  margin: 0;
```